### DISCUSSION TITLE
Chinese language font support

### DISCUSSION THREAD

**Discussion by vectorliu** [posquit0/Awesome-CV#522] [Q&A]
What is the fastest way to configure the template to support Chinese language? I figured I might have to modify the awesome-cv.cls file. So can it be:

1. Add required fonts to the fonts/ directory;

2. Change the font configuration in the cls file.

However, it is not that elegant since in this way I have to change the cls file again if I want to use the original English language font. So is there a more elegant way? Sorry if the question is naive. I am just new to Latex.

**Comment by johannesbottcher** [comment]
Change the font setting in your main file.
Reading an introduction seems to be a good idea as well.

Am 24.02.2017 06:18 schrieb "vectorliu" <notifications@github.com>:

**Comment by vectorliu** [comment]
@johannesbottcher ,did you mean the following changes made to the main file:

\renewcommand*{\footerfont}{\Kaiti}
\renewcommand*{\bodyfont}{\Kaiti}
\renewcommand*{\bodyfontlight}{\STFangsong}

Just tried and it induced a lot more font-related errors.

**Comment by brianhu-tw** [comment]
I use xeCJK package to build my Chinese cv:

\usepackage{xeCJK}

add this line into the fontawsome.sty file

**Reply by DevRuibin** [reply]
This method doesn't work globally if you have multiple files.  Check [this one by @Littlesqx ](https://github.com/posquit0/Awesome-CV/discussions/522#discussioncomment-9364982)

**Comment by vectorliu** [comment]
Thanks@BrianHuGit. Problem solved.

**Comment by johannesbottcher** [comment]
Guys, you can't just change a file that you don't own. There is a license that needs to be respected. 
Add the package to your main file.

**Comment by vectorliu** [comment]
Sorry if it breaks some license. We will add this package in the main file, which solves the problem anyway. Thanks for your kindly remind.

**Comment by 77QingLiu** [comment]
@BrianHuGit , where can I find `fontawsome.sty`?

**Comment by sschyu** [comment]
> @BrianHuGit , where can I find `fontawsome.sty`?

I added the line just after the \documentclass preamble line.
But it doesn't apply the font to the whole page.
I don't know how.

**Comment by Littlesqx** [comment]
> @BrianHuGit , where can I find `fontawsome.sty`?

add into `awesome-cv.cls` can also work @77QingLiu

**Comment by linjh1118** [comment]
> @BrianHuGit , where can I find `fontawsome.sty`?

I find  `fontawsome.sty` in https://github.com/plorcupine/latex-fontawesome. Thanks `latex-fontawesome`

**Comment by ravijo** [comment]
@vectorliu 
@brianhu-tw 
@DevRuibin 
@77QingLiu 
@SiqingYu 

It’s an old thread, but if you’re still looking for a working template, you can find it at [ravijo/Awesome-CV](https://github.com/ravijo/Awesome-CV). Note that it uses `\usepackage{xeCJK}`, and sample TeX files in Japanese are provided; these can be easily adapted for Chinese as needed. Feel free to use it. Thank you.
